client-app/pages/search/search.vue

```diff
@@ -119,6 +120,13 @@ const searchParams = computed<ProductsSearchParams>(() => ({
   keyword: keywordQueryParam.value,
 }));
 
+const productsRoutes = computed(() =>
```

Comment:
Why not to move this snippet to shared file? We reuse this code in every product-related route.